src/main/java/graphql/validation/ValidationError.java

```diff
@@ -15,11 +18,12 @@
 import java.util.stream.Collectors;
 
 @PublicApi
+@NullMarked
 public class ValidationError implements GraphQLError {
 
     private final List<SourceLocation> locations = new ArrayList<>();
-    private final String description;
-    private final ValidationErrorClassification validationErrorType;
+    private final @Nullable String description;
+    private final @Nullable ValidationErrorClassification validationErrorType;
```

Comment:
TODO for me: check alongside other GraphQLError classes if we want to keep the description nullable

Double check ValidationErrorClassification - ErrorClassification is not nullable in GraphQLError